index.html

```diff
@@ -269,24 +276,9 @@
 						<p><span class="fragment">Assign pull request to developer; review</span></p>
 						<p><span class="fragment">Both positive: merge the pull request</span></p>
 					</section>
-				</section>
-
-				<section>
-					<section>
-						<h1>Improving openness of our own projects</h1>
-					</section>
-					<section>
-						<h1>TODO</h1>
-					</section>
-				</section>
-				
-				<section>
-					<section>
-						<h1>GitHub integrations</h1>
-					</section>
 
 					<section>
-						<h1>Board</h1>
+						<h1>Process visualization</h1>
```

Comment:
Updated the bullet points as you suggested. I've kept the first slide before the screenshot, since the first two bullet points are things I'd say before showing the board, but I've moved the 3. and 4. point about waffle to a separate slide after the screenshot. See:

https://hbz.github.io/swib16-workshop/#/7/12